<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLineEdit" name="sRoot">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="tRoot">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>110</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Root:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>110</y>
     <width>47</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Root:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>90</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Source Joints</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>90</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Target Joints</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>250</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Up</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>280</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="downButton_2">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>310</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Down</string>
   </property>
  </widget>
  <widget class="QPushButton" name="transferButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>420</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Transfer Animation</string>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>170</y>
     <width>161</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget_2">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>171</y>
     <width>151</width>
     <height>231</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
